ccx_messaging/publishers/idp_publisher.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2024 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may naot use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module implementing a IDP publisher to Kafka topic."""
+
+
+import json
+from typing import Dict
+import logging
+from ccx_messaging.publishers.kafka_publisher import KafkaPublisher
+
+LOG = logging.getLogger(__name__)
+
+
+class IDPPublisher(KafkaPublisher):
```

Comment:
I would like to find a different name for the file and the class. In the Internal Data Pipeline we can have several publishers, so probable this name is misleading.

Maybe something related to "SyncedArchivePublisher" for example could be better IMHO